### ISSUE TITLE
i updated to v2.2.0 and none of my games work anymore

### ISSUE THREAD

**Issue by leehelck** [ModOrganizer2/modorganizer#797]
## The problem:
Briefly describe the issue you are experiencing. Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our [discord channel](https://discord.gg/5tCqt6V)

## Environment:

* Mod Organizer Version that exhibits the issue:
* Last Mod Organizer Verison that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer:

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

## Link to Mod Organizer logs:
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder". You should close MO2 before sending or examining the logs as they are finished writing only after the program is closed.

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your USVFS logs, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by Al12rs** [comment]
I don't really know what we are supposed to do with a report like this. I can tell you that you are the only one to have reported anything wrong with updating, so this isn't a common occurrence. Please provide some more information such as, what does "do not work anymore" mean? Doe games not start? Do they crash while playing? Do they work outside Mo2? Do you mean also completely unrelated games?
What type of setup did you have etc.
Most of the useful info is in your logs so if you can provide those as instructed by the template that could help a long wai for us to be able to help you.

**Comment by leehelck** [comment]
I updated MO2 to v2.2.0 on each of my "portable" installations for FO3, FNV, and TES4 and it completely ruined them. the games launch vanilla with no problems at all. FO3 won't launch through MO2 at all, FNV launches but has a weird issue where the game is darker than usual, and TES4 launches but tells me that I have missing files when I try to load my most recent save. none of the load orders have changed. I haven't messed with those load orders in at least half a year as they were stable. in addition I have warnings in each instance of MO2 about "incompatible" mods, yet these load orders worked fine before the update. I am also getting the same warning in FO4. here are the logs:

[mo_interface.log](https://github.com/ModOrganizer2/modorganizer/files/3389165/mo_interface.log)
[mo_interface.log](https://github.com/ModOrganizer2/modorganizer/files/3389167/mo_interface.log)
[mo_interface.log](https://github.com/ModOrganizer2/modorganizer/files/3389170/mo_interface.log)
[mo_interface.log](https://github.com/ModOrganizer2/modorganizer/files/3389174/mo_interface.log)

FO4 works fine even with the warnings, so i'm not really concerned with that. Skyrim LE and SSE are also working without issue. is the reason why those games aren't working because those games are older? I had to do some workarounds to get them to work initially through MO2, although I don't exactly remember how I did them as it was so long ago when I created those load orders. another thing: sorry about the vague initial post. I didn't realize it posted as I was looking into the problems at hand and also testing my other games, checking file integrity, etc.

**Comment by Al12rs** [comment]
I'm guessing that something went wrong during installation, most likely either windows or your AntiVirus eating or blocking the MO2 files making it not actually virtualize. Check your AV and make sure to put exclusions for all the mo2 folders. Then try reinstalling MO2 on top of the installations. I would advise using the installer as it should clean up some of the older files that are no longer needed. You can get the installer from the MO2 Nexus page (SSE nexus).

**Comment by Silarn** [comment]
You also still failed to provide usvfs logs still, which would likely verify what Al said.

Either Windows is blocking files in the archive (which means you need to unblock it before extracting) or your antivirus is quarantining or deleting usvfs executables/libraries.

**Comment by leehelck** [comment]
I tried whitelisting the folder in my AV, and re-installing with the installer to no avail. I have uninstalled the games in question and am going to re-install them along with a new fresh installation of MO2. I was planning on moving those games and their corresponding MO2 install folders to a different hard drive anyway, it just turns out it's going to happen sooner rather than later lol. thanks for trying to help me out.
